app/assets/javascripts/application.js

```diff
@@ -1 +1,21 @@
-//= require header-footer-only
+/* global $ */
+
+//= require govuk_publishing_components/dependencies
+
+//= require govuk_publishing_components/lib
```

Comment:
Is everything in `lib` needed for the error pages? Based on a quick glance, you might only need `track-click` and `cookie-functions`?

Reply:
From a bit of trial and error, I think that the only required bits are:

 - `cookie-functions`
 - `header-navigation`
 - `track-click`

📉 👍